frontend/express/public/templates/user-activity/UserActivity.html

```diff
@@ -0,0 +1,124 @@
+<script type="text/x-template" id="user-activity">
+    <div v-bind:class="[componentId]">
+        <cly-header>
+            <template v-slot:header-left>
+                <h2> {{i18n('user-activity.title')}} 
+                </h2>
+                <span class="el-icon-info" style="margin-left:10px" v-tooltip.top-center="description">
+                </span>
+            </template>
+        </cly-header>
+        <cly-main>
+            <user-activity-filter> </user-activity-filter>
+            <user-activity-bar-chart> </user-activity-bar-chart>
+            <user-activity-table> </user-activity-table>
+        </cly-main>
+    </div>
+</script>
+
+<!-- NOTE-LA: <user-activityfilter> is a temporary created component until query filter with modal becomes available-->
+<script type="text/x-template" id="user-activity-filter">
+    <cly-section style="min-height:150px">
+        <div class="bu-columns bu-is-gapless bu-is-mobile bu-is-centered" style="padding-top:1%">
+            <div class="bu-column bu-is-12-tablet bu-is-9-desktop bu-is-7-widescreen bu-is-6-fullhd">
+                <cly-qb-segmentation
+                    :allow-breakdown="false"
+                    v-model="userActivityFilters"
+                >
+                </cly-qb-segmentation>
+            </div>
+        </div>
+        <div class="bu-columns bu-is-gapless bu-is-mobile bu-is-centered" style="padding-bottom:1%">
+            <div class="bu-column bu-is-12-tablet bu-is-9-desktop bu-is-7-widescreen bu-is-6-fullhd">
+                <div class="bu-columns">
+                    <div class="bu-column bu-is-1">
+                        <el-button type="info" plain>Cancel</el-button>
+                    </div>
+                    <div class="bu-column bu-is-1">
+                        <el-button type="success" @click="onApplyFilter">Apply filter</el-button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </cly-section>
+</script>
+
+<script type="text/x-template" id="user-activity-bar-chart">
+    <cly-section>
+        <cly-chart-bar :option="userActivityOptions" style="min-height: 400px;" :autoresize="true" v-loading="isLoading"></cly-chart-bar>
```

Comment:
charts accept height as a prop, so instead of using style, you can pass :height="400". 
Note: height should be a Number.
autoresize is also true by default so no need to set it.